tests/test_json_s3_to_parquet.py

```diff
@@ -0,0 +1,425 @@
+import json
+import os
+import time
+import boto3
+import pandas
+import pytest
+from awsglue.context import GlueContext
+from pyspark.sql.session import SparkSession
+from src.glue.jobs import json_s3_to_parquet
+# requires pytest-datadir to be installed
+
+
+class TestJsonS3ToParquet:
+
+    @pytest.fixture(scope="class")
+    def glue_database_name(self, namespace):
+        return f"{namespace}-pytest-database"
+
+    @pytest.fixture(scope="class")
+    def glue_nested_table_name(self):
+        return "dataset_pytest_nested_table"
+
+    @pytest.fixture(scope="class")
+    def glue_flat_table_name(self):
+        return "dataset_pytest_flat_table"
```

Comment:
Nit: These can technically be part of a `setup_class`  function.

```
def setup_class(self, namespace):
    self.glue_database_name = f"{namespace}-pytest-database"
    ....
```

You can then call these in the other functions without everything being fixtures.

Reply:
I prefer using fixtures.